index.html

```diff
@@ -231,21 +243,21 @@ <h4>LSIF Indexers</h4>
 				<th>Java</th>
 				<td><a href="https://sourcegraph.com/">Sourcegraph</a></td>
 				<td class="repo nowrap"><a href="https://github.com/sourcegraph/lsif-java">github.com/sourcegraph/lsif-java</a></td>
-				<td class="table-success"><i class="material-icons">check</i></span></td>
-				<td class="table-success"><i class="material-icons">check</i></span></td>
-				<td class="table-success"><i class="material-icons">check</i></span></td>
-				<td class="table-success"><i class="material-icons">check</i></span></td>
+				<td class="table-success"><i class="material-icons">check</i></td>
+				<td class="table-success"><i class="material-icons">check</i></td>
+				<td class="table-success"><i class="material-icons">check</i></td>
+				<td class="table-success"><i class="material-icons">check</i></td>
 				<td class="table-warning"></td>
 				<td><span class="badge badge-warning">Development</span><span class="badge badge-danger">Unstable</span></td>
-				<td>Supports Maven and Gradle.</td>
+				<td>Supports Maven, Gradle and SBT.</td>
```

Comment:
"SBT" is only used by people who never use sbt themselves 😛 
```suggestion
				<td>Supports Maven, Gradle and sbt.</td>
```

Reply:
I have exposed myself :gohno: